eq-author/cypress/integration/authenticated/builder_spec.js

```diff
@@ -510,6 +510,16 @@ describe("builder", () => {
     });
   });
 
+  it("Can add a new question page", () => {
```

Comment:
Can we move this test into the other add page test and explain in that test title that we have two different methods of adding a page?

Reply:
I've moved the new test into the existing add page test